presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaPageSourceProvider.java

```diff
@@ -114,6 +117,8 @@ private InternalTable getInternalTable(ConnectorSession connectorSession, Connec
 
     public InternalTable getInformationSchemaTable(Session session, String catalog, InformationSchemaTable table, Set<QualifiedTablePrefix> prefixes, OptionalLong limit)
     {
+        log.debug("Building information schema table (queryId=%s; catalog=%s; table=%s; prefixes=%s)", session.getQueryId(), catalog, table, prefixes);
```

Comment:
log limit too

is query id useful?

Reply:
> is query id useful?

It's easier to track exact query that caused the log message